ACE/ace/SOCK_Dgram_Mcast.h

```diff
@@ -376,15 +376,15 @@ class ACE_Export ACE_SOCK_Dgram_Mcast : public ACE_SOCK_Dgram
   /// Empty the dynamic subscription list.
   int clear_subs_list ();
 
+  /// Multicast address to which local send() methods send datagrams.
```

Comment:
Don't reorder member declarations as that will trigger compile warnings on the ordering of these members in the constructors

Reply:
I had to, because I moved two members into the 'protected' area. Updated ctor in SOCK_Dgram_Mcast.cpp